contents/tutorials/ghost-analytics.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: "Quick start: Ghost analytics and session replay"
+date: 2024-10-11
+featuredTutorial: false
+author:
+  - danilo-campos
+tags:
+  - product analytics
+  - web analytics
+  - session replay
+---
+
+import Snippet from "../docs/integrate/snippet.mdx"
+
+With all the drama in the world of WordPress, it’s a great time to explore Ghost, a robust blogging and newsletter CMS. You can [self-host](https://ghost.org/docs/install/) Ghost on your own server, or lean on managed hosting from [Ghost.org](https://ghost.org).
+
+However you host, adding PostHog for [web analytics](/docs/web-analytics), [product analytics](/docs/product-analytics), and [session replay](/docs/session-replay) is a quick process that gives you immediate insight about how people are finding your site, enjoying your writing and exploring your archives. PostHog accounts are free, so sign up and follow along.
```

Comment:
1. We are oxford comma people :)
2. We generally don't refer to "PostHog accounts". Instances, projects, or just "PostHog" is fine: 
```suggestion
However you host, adding PostHog for [web analytics](/docs/web-analytics), [product analytics](/docs/product-analytics), and [session replay](/docs/session-replay) is a quick process that gives you immediate insight about how people are finding your site, enjoying your writing, and exploring your archives. PostHog is free, so sign up and follow along.
```